Nodejs/Product/Nodejs/Debugger/NodeDebugProvider.cs

```diff
@@ -113,31 +113,6 @@ private void SetupDebugTargetInfoForWebkitV2Protocol(ref VsDebugTargetInfo vsDeb
         {
             // todo: refactor the debugging and process starting so we can re-use
 
-            var setupConfiguration = new SetupConfiguration();
```

Comment:
Does this mean we can remove a reference?

Reply:
Done